softvis3d-frontend/src/classes/Metric.ts

```diff
@@ -1,13 +1,16 @@
 export default class Metric implements SelectOptionValue {
 
-    public id: string;
-    public name: string;
-    public type: MetricType;
+    public description: string;
 
-    constructor(id: string, type: MetricType, name: string) {
+    private id: string;
+    private name: string;
+    private type: MetricType;
```

Comment:
I also saw this during the little refactoring, but thought that we will need it during the backend refactoring and left it as is. But I'm not sure, so i will delete it.